src/Ztm.WebApi/TransactionConfirmationWatchingWatchStatus.cs

```diff
@@ -0,0 +1,10 @@
+namespace Ztm.WebApi
+{
+    public enum TransactionConfirmationWatchingWatchStatus
+    {
+        Pending = 0,
+        Success = 1,
+        Rejected = 2,
+        Error = 3
```

Comment:
Seems like no one using this and no plan to implement it yet. We should remove it for now.